docs/opcodes/53.mdx

```diff
@@ -3,17 +3,19 @@ fork: Frontier
 group: Stack Memory Storage and Flow Operations
 ---
 
+*Index 1 is top of the stack. See [PUSH](/#60).*
+
 ## Stack input
 
 0. `offset`: offset in the [memory](/about) in bytes.
 1. `value`: 1-byte value to write in the [memory](/about) (the least significant byte of the 32-byte stack value).
 
 ## Examples
 
-| Input 1 | * | Input 2 |
-|------:|--:|--:|
-| `0` | * | `1` |
-| `0xFFFF` | * | `0xFF` |
+| * | Input 1 | * | * | Input 2 |
+|--:|--------:|--:|--:|--------:|
+| `1` | `0` | * | `1` | `1` |
+| `2` | `0xFFFF` | * | `2` | `0xFF` |
 
 | Memory before |
 |--------------:|
```

Comment:
Nit: maybe we could also add the memory slot index for consistency, but totally fine without too

Reply:
There is no memory slot, only storage slot. Memory is byte indexed. Not sure how we can make that more clear, showing it byte by byte is not a nice solution I think.

Reply:
that's true, I agree